app/controllers/subscriptions_controller.rb

```diff
@@ -30,6 +29,14 @@ def show
 
   private
 
+  def create_subscription(subscription_params, iugu_id)
```

Comment:
Since you already have a method called `subscription_params`, I'd recommend naming the method parameter differently. That way you don't "shadow" the controller method.

For this case, `attributes` seems a good name to me.